src/blog/lagom-1-3-6.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Lagom 1.3.6 is released!
+date: 2017-07-17
+author_github: TimMoore
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.3.6
+---
+
```

Comment:
I would re-structure the first two paragraphs into a bulleted list, something like:

For the specific changes in Lagom 1.3.6, see (link to either the changelog.html or the section below, see my comments below) As an overview, this release contains the following:
- Small improvements from the preview release of Lagom 1.4.0, which have been backported to this release.
- Small bug fixes and minor enhancements 
- Deprecated APIs to fix typos and inconsistencies.
- Improvement in event publishing to Kafka by ensuring that topic producers are always restarted after failure (with exponential back off) (And, I would add whether it happens without the developer having to do anything)